src/context-parser-handlebars.js

```diff
@@ -76,12 +82,18 @@ function ContextParserHandlebars(config) {
     /* the flag is used to strict mode of handling un-handled state, defaulted to false */
     this._config._strictMode = (config.strictMode === true);
 
+    /* the flags are used to set the partial handling */
+    this._config._enablePartialCombine = (config.enablePartialCombine !== false);
+
     /* save the char/line no being processed */
     this._charNo = 0;
     this._lineNo = 1;
 
     /* context parser for HTML5 parsing */
     this.contextParser = parserUtils.getParser();
+
+    /* internal file cache */
+    this._partialsCache = Object.create(null);
```

Comment:
TODO: find a way to integrate _partialsCache with what retrieved from Handlebars.registerPartial()